stubstub/modelize/link.ts

```diff
@@ -2,41 +2,105 @@
 import uuid from 'uuid/v4';
 import * as graphlib from 'graphlib';
 
+import { Model, ModelCtor } from 'sequelize';
 import { sequelizeModels } from './models';
 
-const modelByName = (modelName) => {
+const modelByName = (modelName: string) => {
 	const model = sequelizeModels[modelName];
 	if (!model) {
 		throw new Error(`Modelize cannot find a model named ${modelName}`);
 	}
 	return model;
 };
 
-const walkAst = (ast, parameters) => {
-	const definitionsByBoundName = {};
-	const definitions = [];
-	const resolveIdentifiersCallbacks = [];
+export type ModelDefinition = {
```

Comment:
these types are inferred from the parser, which i did not change.
I think they are accurate!